db/schema.rb

```diff
@@ -142,9 +142,11 @@
   end
 
   create_table "zip_code_tabulation_areas", force: :cascade do |t|
-    t.text "zipcode", null: false
-    t.float "lat", null: false
-    t.float "lng", null: false
+    t.text "zipcode"
```

Comment:
I'm not sure why this table in schema.rb changed. I didn't do anything to this table.